CONTRIBUTING.md

```diff
@@ -82,6 +172,30 @@ cd code
 yarn build --prod --watch angular core addon-docs
 ```
 
-## Contributing to Storybook
+### Running against different sandbox templates 
+
+You can pick a specific template to use as your sandbox by running `yarn task`, which will prompt you to make further choices about which template you want and which task you want to run.
+
+## Troubleshooting 
+
+### The initialization process throws an error 
+
+If you run `yarn start` and encounter the following error, try rerunning `yarn start` a second time: 
+
+```shell 
+> NX ENOENT: no such file or directory, open 'storybook/code/node_modules/nx/package.json' 
+```
+
+### Storybook doesn't detect changes in the codebase 
+
+If you are a Storybook contributor and still experience issues, it is recommended that you verify your local Storybook instance for any unintentional local changes. To do this, you can use the following command: 
+
+```shell 
```

Comment:
@usrrname also one minor item for consistency, could you use `shell` for all shell-based commands to align with the rest of the documentation?